packages/web/components/buttons/button.tsx

```diff
@@ -141,8 +141,27 @@ export const buttonCVA = cva(
           "rounded-xl",
           "p-2",
         ],
+        "staking-learn-more-modal": [
+          "border-2",
+          "border-osmoverse-800",
+          "border-osmoverse-700",
+          "bg-osmoverse-700",
+          "hover:border-osmoverse-825",
+          "hover:bg-osmoverse-825",
+          "rounded-xl",
+        ],
+        "staking-learn-more-no-modal": [
+          "border-2",
+          "border-osmoverse-700",
+          "border-osmoverse-800",
+          "bg-osmoverse-800",
+          "hover:border-osmoverse-700",
+          "hover:bg-osmoverse-700",
+          "rounded-xl",
+        ],
```

Comment:
IMO it's a leaky abstraction to add staking logic into the reused button component. Could we remove references to staking and perhaps name it in a way that describes how it looks?